### PR TITLE
ODIM-5511: Sub task resource updated as per comments from OCP profile validation

### PR DESCRIPTION
**Pull request by chauberahul1993**
ODIM-5511: Sub task resource updated as per comments from OCP profile validation

### CODE REVIEW

**Review by amar-shalgar — COMMENTED** [review]
Duplicate PR

File: svc-task/tresponse/task.go [hunk 1]
```diff
@@ -43,7 +44,7 @@ type Task struct {
 	StartTime       time.Time   `json:"StartTime"`
 	EndTime         time.Time   `json:"EndTime,omitempty"`
 	TaskStatus      string      `json:"TaskStatus"`
-	SubTasks        string      `json:"SubTasks,omitempty"`
+	SubTasks        ListMember  `json:"SubTasks,omitempty"`
```
Comment (jeevan-kamkar) [inline svc-task/tresponse/task.go:47]: since it's a structure change it to the pointer so that if ti's nil omitempty will be applicable

**Review by jeevan-kamkar — CHANGES_REQUESTED** [review]
Address the requested changes

File: svc-task/tresponse/task.go (hunk 1 above)
Comment (chauberahul1993) [reply svc-task/tresponse/task.go]: Done

**Review by shivacharanms — APPROVED** [review]

**Review by amar-shalgar — APPROVED** [review]

**Pull request merged**